### ISSUE TITLE
Not able to farm Pokéstops? - Out of range?

### ISSUE THREAD

**Issue by DUSKvsDAWN** [finndev/PokeBuddy#1211]
This might be a new version of a softban, but I'm not sure. Cause I can still catch Pokémon, which you normally can't when you are softbanned. And this "possible softban" has been going on way longer than my normal softbans last. Normally they are like 3 hours max. It's been around 8 hours now.. Anyone know if this indeed IS a softban, and when it could go away? Or is it just a bug?

**Comment by rmarquez5** [comment]
Yeah, some sites report this behavior as another type of softban. Hopefully it lifts itself off you soon.

**Comment by DUSKvsDAWN** [comment]
Yeah, I hope so too. I read that it might be a softban, I just wondered if there was more info on it yet..

**Comment by Demolition1986** [comment]
Well i got same softban as you, and like 10 or 12h you are free. How many hours u still with that?

**Comment by DUSKvsDAWN** [comment]
like i said im around 8 or 9 hours in... i hope it vanishes soon. i dont even have pokeballs left.

**Comment by Demolition1986** [comment]
i have more than 300 poke, but i got bored xP

**Comment by DUSKvsDAWN** [comment]
Still banned, it seems. It's taking forever.
